pkg/liqo-controller-manager/storageprovisioner/remoteprovisioner.go

```diff
@@ -13,3 +13,145 @@
 // limitations under the License.
 
 package storageprovisioner
```

Comment:
Would personally maintain the structure of the other components of the virtual kubelet, with function in the forge package responsible for forging the local/remote objects and the creation/update logic within the corresponding reflector. Decoupling makes it also easier to write tests. Mostly matter of taste though, feel free to ignore.